README.md

```diff
@@ -1,38 +1,160 @@
+| `logoTitle`     | `String`  | Alt text for the logo                             | -             |
+| `showSearchBar` | `Boolean` | Sets whether the search bar is visible or not     | true          |
+| `showLogin`     | `Boolean` | Sets whether the login button is displayed or not | true          |
+
+Also, you can configure the blocks [telemarketing](https://github.com/vtex-apps/telemarketing), [logo](https://github.com/vtex-apps/store-components/blob/master/react/components/Logo/README.md), [login](https://github.com/vtex-apps/login), [category-menu](https://github.com/vtex-apps/category-menu), [search-bar](https://github.com/vtex-apps/store-components/blob/master/react/components/SearchBar/README.md) and [menu-link](https://github.com/vtex-apps/menu) defined on header.
+
+### Styles API
+
+This app provides some CSS classes as an API for style customization.
+
+To use this CSS API, you must add the `styles` builder and create an app styling CSS file.
+
+1. Add the `styles` builder to your `manifest.json`:
+
 ```json
- "store/header": {
-      "component": "vtex.dreamstore-header/index"
- }
+  "builders": {
+    "styles": "1.x"
+  }
 ```
 
-See an example at [Dreamstore](https://github.com/vtex-apps/dreamstore-theme/blob/master/pages/pages.json#L7) and [Store](https://github.com/vtex-apps/store/blob/master/react/StoreTemplate.js#L14) apps
+2. Create a file called `vtex.header.css` inside the `styles/css` folder. Add your custom styles:
+
+```css
+.container {
+  margin-top: 10px;
+}
+```
+
+#### CSS namespaces
+
+Below, we describe the namespaces that are defined in the header.
+
+| Class name           | Description                                     | Component Source                            |
+| -------------------- | ----------------------------------------------- | ------------------------------------------- |
+| `container`          | The main container of header                    | [index](/react/index.js)                    |
+| `leanMode`           | The main container of header on lean mode       | [index](/react/index.js)                    |
+| `topMenuContainer`   | The container of `fixed` top menu               | [index](/react/components/FixedContent.js)  |
+| `topMenuLogo`        | The container of logo in `fixed` top menu       | [Popup](/react/components/Logo.js)          |
+| `topMenuSearchBar`   | The container of search bar in `fixed` top menu | [Popup](/react/components/SearchBar.js)     |
+| `topMenuIcons`       | The container of icons on `fixed` top menu      | [Sidebar](/react/components/Icons.js)       |
+| `topMenuCollapsible` | The container of `collapsible` top menu         | [Sidebar](/react/components/Collapsible.js) |
+
+## Troubleshooting
+
+You can check if others are passing through similar issues [here](https://github.com/vtex-apps/minicart/issues). Also feel free to [open issues](https://github.com/vtex-apps/minicart/issues/new) or contribute with pull requests.
```

Comment:
Fix the links, you are use the minicart repository instead of the store-header one.